modules/osdk-hh-defining-go-api.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-hh-defining-go-api_{context}"]
+= Defining the API
+
+Define the API for the Memcached custom resource (CR).
+
+Represent this Go API by defining the `MemcachedBackup` type which will have a `MemcachedBackupSpec.Size` field to set the quantity of Memcached backup instances (CRs) to be deployed, and a `MemcachedBackupStatus.Nodes` field to store a CR's pod names.
```

Comment:
What does _represent_ mean here? 
s/type which/type, which  --or-- type that 
Or: _defining the MemcachedBackup type with a MemcachedBackupSpec.Size field set the quantity....